src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+<documentation title="Doc Comment">
+    <standard>
+    <![CDATA[
+    Enforces rules related to the formatting of DocBlocks ("Doc Comments") in PHP code.
+
+    DocBlocks are a special type of comment that can provide information about a structural element. In the context of DocBlocks, the following are considered structural elements: require(_once) and include(_once) statements, class, interface, trait, function, property, constant and variable declarations.
```

Comment:
Any particular reason not to mention enums ?

Also, would it make more sense to have the (less commonly commented on) `require(_once) and include(_once) statements` bit at the end ? That would show the structures which more commonly have DocBlocks first.

Reply:
Good points! I pushed a new commit implementing those two suggestions.